server/routes/referrals/complexityLevelForm.ts

```diff
@@ -0,0 +1,29 @@
+import { Request } from 'express'
+import { DraftReferral } from '../../services/interventionsService'
+import { ComplexityLevelError } from './complexityLevelPresenter'
+
+export default class ComplexityLevelForm {
+  private constructor(private readonly request: Request) {}
+
+  static async createForm(request: Request): Promise<ComplexityLevelForm> {
+    return new ComplexityLevelForm(request)
+  }
+
+  get paramsForUpdate(): Partial<DraftReferral> {
+    return {
+      complexityLevelId: this.request.body['complexity-level-id'],
+    }
+  }
+
+  get isValid(): boolean {
+    return this.request.body['complexity-level-id'] != null
```

Comment:
this is what we just talked about right? looks like we are already doing this validation then?

Reply:
Yep - we were wondering whether to get rid of this here, but I'll leave it in now